src/Schema/Events/Tap.ts

```diff
@@ -1202,10 +1203,8 @@ export interface TappedNewsSection {
  *  }
  * ```
  */
-export interface TappedNotification {
-  action: ActionType.tappedNotification
-  context_module: ContextModule
-  context_screen_owner_type: ScreenOwnerType
+export interface TappedTaskGroup extends TappedEntityGroup {
+  action: ActionType.tappedTaskGroup
   destination_path: string
   notification_id: string
   notification_category: string
```

Comment:
Should we also rename these payload keys to `task_[field]`? Is a task's `category` expected to be the same value as the task's `type` in Gravity?

I don't see category in the backend data modeling but a [task's type](https://github.com/artsy/gravity/blob/95e9760d3512a8b8ad1d250c6b7d141938579196/app/models/domain/task.rb#L2) can currently be one of send_wire, outbid, counteroffer, or id_verification.

Reply:
I agree with the change to rename to `task_[field]`, overlooked that when fixing the naming.

For your second question, do you suggest renaming the (now) `task_category` into `task_type`, or to deprecate `task_category` and instead make use of `type` inherited from TappedEntityGroup? (Would the TappedEntityGroup type link to the `TYPES` referenced in [task.rb](https://github.com/artsy/gravity/blob/95e9760d3512a8b8ad1d250c6b7d141938579196/app/models/domain/task.rb)?

Reply:
I think it's fine to extend the TappedEntityGroup interface with two new custom fields to track task metadata (id and type). I wouldn't reuse the existing `type` (not prefixed) property on TappedEntityGroup as that refers to something else.

As far as I can tell, the Task model defined in Gravity doesn't yet have a "category" concept. If we're instead expecting the internal "task type" value here, we can use a similar property name to avoid conflicts in the future if we ever want to introduce categories to the Task model.

So the full interface might look something like this:

```ts
export interface TappedTaskGroup extends TappedEntityGroup {
  action: ActionType.tappedTaskGroup
  destination_path: string
  task_id: string
  task_type: string
Member

```

Reply:
I see, this makes sense. I misunderstood what you were trying to communicate before, and agree with this change. Thank you for your help and the completed example!